.github/workflows/tests_secondary.yml

```
@@ -132,7 +132,7 @@ jobs:
         DEBUG: pw:install
     - run: npm run build
     - run: npx playwright install-deps
-    - run: xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" -- bash installation-tests/installation-tests.sh
+    - run: xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" -- bash installation-tests/run_all_tests.sh
```

```suggestion
    - name: Running tests
      run: xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" -- bash installation-tests/run_all_tests.sh
```

Make it more clear that this one is the important one.